src/Modal.js

```diff
@@ -428,12 +440,21 @@ class Modal extends React.Component {
     }
   }
 
+  handleDocumentKeyDown = (e) => {
+    if (this.props.keyboard && e.keyCode === 27 && this.isTopModal()) {
```

Comment:
can we do `e.key === 'Escape'`? (and below too)

i know this is an old issue, but it'd clean things up a tiny bit